src/pages/Profile.tsx

```diff
@@ -125,13 +133,7 @@ const Profile = () => {
             dispatch(openDialog({ type: DialogTypes.Onboarding }));
           },
         },
-  ].filter(isTruthy) as {
-    key: string;
-    label: string;
-    icon: IconButtonProps;
-    href?: string;
-    onClick?: () => void;
-  }[];
+  ].filter(isTruthy);
```

Comment:
[q] if the actions array is hardcoded, why do we need to filter on Truthiness?

Reply:
@yogurtandjam the `isTruthy` method will filter out any actions that are undefined (in the past we may have had actions that were conditional) and persist the `Actions[]` type. With the current list of actions we could safely remove the filter.

Reply:
gotcha. would that mean at some point someone might have just added `undefined` into the array? 🤔

Reply:
Something like adding `shouldAdd && {...},` which may result in an array `[Action, Action, false, Action, undefined]`. Using `isTruthy` would filter the array and preserve the `Action[]` type